options/options.c

```diff
@@ -519,7 +519,7 @@ const m_option_t mp_opts[] = {
 
     // -1 means auto aspect (prefer container size until aspect change)
     //  0 means square pixels
-    OPT_ASPECT("video-aspect", movie_aspect, UPDATE_IMGPAR, -1.0, 10.0),
```

Comment:
Doesn't this and the OPT_REPLACE below means the behavior of ``video-aspect`` as visible to the user changes, which would break compatibility? I didn't try it myself, and what actually happens probably depends much on the property/option bridge in presence of renamed options, which isn't really a rabbit hole I want to get down.

Reply:
Oh, yes, that doesn't interact at all. Good catch. This actually segfaults.

Reply:
Decided to just have the property directly feed through to video-aspect-override without going via the deprecated option bridge, and print a deprecation warning via some dumb hack.

The plan is to rewrite this function completely once the deprecation period is over, and script authors have had time to adjust. In the meantime, the bug will be fixed for users.